packages/circuits/test/proveReportNonNullifier.test.ts

```diff
@@ -97,77 +95,51 @@ describe('Prove report negative reputation in Unirep Social-TW', function () {
             chainId
         )
 
-        const circuitInputs = genReportNegRepCircuitInput({
+        const circuitInputs = genReportNonNullifierCircuitInput({
             reportedEpochKey,
             identitySecret,
             reportedEpoch,
             currentEpoch,
             currentNonce,
             attesterId,
             chainId,
-            type,
         })
 
-        const { isValid, publicSignals } = await genProofAndVerify(
-            circuit,
-            circuitInputs
-        )
-        expect(isValid).to.be.true
-        const epochKey = publicSignals[0]
-        expect(epochKey.toString()).to.be.equal(
-            utils
-                .genEpochKey(
-                    BigInt(identitySecret),
-                    attesterId,
-                    currentEpoch,
-                    currentNonce,
-                    chainId
-                )
-                .toString()
-        )
-        // decode other data
-        const controlData = decodeEpochKeyControl(BigInt(publicSignals[1]))
-        expect(controlData.epoch.toString()).to.be.equal(
-            currentEpoch.toString()
-        )
-        expect(controlData.attesterId.toString()).to.be.equal(
-            attesterId.toString()
-        )
-        expect(controlData.chainId.toString()).to.be.equal(chainId.toString())
-
-        // we don't reveal the nonce, so this is equal to BigInt(0)
-        expect(controlData.nonce.toString()).to.be.equal('0')
+        try {
+            const { isValid } = await genProofAndVerify(circuit, circuitInputs)
+            expect(isValid).to.be.false
+        } catch (error) {
+            console.log('Expected error occurred:\n\n', error)
+        }
```

Comment:
Don't recommend to use try-catch in the test cases. If you still want to write try-catch in the test cases, write some assertions in catch block too.